conf/broker.conf

```diff
@@ -58,6 +58,9 @@ advertisedAddress=
 # Number of threads to use for Netty IO. Default is set to 2 * Runtime.getRuntime().availableProcessors()
 numIOThreads=
 
+# Number of threads to use for orderedExecutor. Default is 8
```

Comment:
Could you please add more description for the ordered executor in the broker? For example, what tasks will he be used to perform, etc?

Reply:
Thanks for your feedback. I have updated some descriptions, please take a look.